tests/UI/specs/UIIntegration_spec.js

```diff
@@ -333,6 +333,20 @@ describe("UIIntegrationTest", function () { // TODO: Rename to Piwik?
         }, done);
     });
 
+    // Do not allow API response to be displayed
+    it('should not allow to widgetize an API call', function (done) {
+        expect.screenshot('widgetize_apidisallowed').to.be.captureSelector('#content', function (page) {
```

Comment:
fixed in https://github.com/piwik/piwik/commit/66511ac4c0086a1887a6119280f9571b643f6a6a